app/components/projects/projects-page.tsx

```diff
@@ -0,0 +1,19 @@
+import ProjectsCarousel, { ProjectsCarouselProps } from "./projects-carousel";
+
+const ProjectsPage = ({ projects }: ProjectsCarouselProps) => {
```

Comment:
nit: we can have this directly in the projects.tsx route, no need to have a component just for the page itself